src/styles/components/_definition.scss

```diff
@@ -1,17 +1,65 @@
 $breakpoint-background-image-size: 75rem;
 
 .cmp-definition {
-  position: relative;
   line-height: 1.75rem;
   color: var(--foreground-color);
+  width: 100%;
+  position: relative;
+  margin: 2.5rem 0 0; // top: 40px; left&right: 0; bottom: 0;
 
-  // the text column
-  &__text {
-    padding: 0;
-    max-width: 60ch;
+  @media (min-width: $breakpoint-scale-up) {
+    margin-bottom: -8.5625rem; // compensates for code example padding
+    background:
+      linear-gradient(to right, var(--background-color) 75%, transparent),
+      url("../../../public/images/definition_text_light.svg");
+    background-size: contain;
+    background-position: center right -3rem;
+    background-repeat: repeat-x;
+
+    @media (prefers-color-scheme: dark) {
+      background: linear-gradient(to right, var(--background-color) 75%, transparent), url("../../../public/images/definition_text_dark.svg");
+      background-size: contain;
+      background-position: center right -2rem;
+      background-repeat: repeat-x;
+    }
+  }
 
-    @media (min-width: $breakpoint-large-screen-860) {
-      padding: 8rem 0;
+  @media (min-width: $breakpoint-background-image-size) {
+    background:
+      linear-gradient(to right, var(--background-color) 50%, transparent),
+      url("../../../public/images/definition_text_light.svg");
+    background-size: contain;
+    background-position: center right -3rem;
+    background-repeat: repeat-x;
+
+    @media (prefers-color-scheme: dark) {
+      background: linear-gradient(to right, var(--background-color) 50%, transparent), url("../../../public/images/definition_text_dark.svg");
+      background-size: contain;
+      background-position: center right -2rem;
+      background-repeat: repeat-x;
+    }
+  }
```

Comment:
Remember that we can delete any of this code that is duplicated from prior media q's thanks to the CSS cascade. A selector will inherit all of the settings 

```css
  @media (min-width: $breakpoint-scale-up) {
    margin-bottom: -8.5625rem; // compensates for code example padding
    background:
      linear-gradient(to right, var(--background-color) 75%, transparent),
      url("../../../public/images/definition_text_light.svg");
    background-size: contain;
    background-position: center right -3rem;
    background-repeat: repeat-x;

    @media (prefers-color-scheme: dark) {
      background-image: url("../../../public/images/definition_text_dark.svg");
      background-position: center right -2rem;
    }
  }

  @media (min-width: $breakpoint-background-image-size) {
    background:
      linear-gradient(to right, var(--background-color) 50%, transparent),
      url("../../../public/images/definition_text_light.svg");
    background-position: center right -3rem;

    @media (prefers-color-scheme: dark) {
      background-image: url("../../../public/images/definition_text_dark.svg");
      background-position: center right -2rem;
    }
  }
  ```

Reply:
Do you mean to change the background-position from -3rem to -2rem on dark mode?

Reply:
> Do you mean to change the background-position from -3rem to -2rem on dark mode?

Yes- the 3rem to 2rem was to try and compensate for slight differences in the SVG. If you flip between light and dark mode, "Definition" doesn't "move" as much with that adjustment.

Reply:
> Do you mean to change the background-position from -3rem to -2rem on dark mode?

I forgot- I have new SVG's for that now, so they do not need to be different any longer. Thanks. 

I did a little refactoring again. Something about the `background` vs `background-image` was giving me some wonky results, where I had to keep those repetitions in. I was getting stuff like this: 
<img width="599" alt="Screen Shot 2022-11-28 at 1 27 08 PM" src="https://user-images.githubusercontent.com/69602589/204357943-b57adc64-a6a3-4be9-be05-41b8286e24b4.png">

Anyways, I think it's better now. I put the `linear-gradient` on the new container, instead of the entire component and that seemed to help the media queries be less repetitive. I was also able to get rid of a media query.

https://github.com/sparkbox/accessible-components/pull/89/commits/896296a115c207cb7b7af20bf15cab72a5121a59